src/AuctionHouseBot.cpp

```diff
@@ -24,70 +22,12 @@
 #include "Player.h"
 #include "WorldSession.h"
 #include "GameTime.h"
+#include "StringConvert.h"
+#include "StringFormat.h"
```

Comment:
```suggestion
#include "DatabaseEnv.h"
#include "StringConvert.h"
#include "StringFormat.h"
```
This fixes merge conflict with master branch